clients/README.md

```diff
@@ -1,8 +1,24 @@
 # Client implementation wrappers
 
-This directory contains some client wrappers: they are executables that provide a specific
-command line interface that the test suite uses. Embedding these wrappers here is likely temporary:
-The intention is that TUF implementations that want to use the test suite will include wrappers like this in
+The conformance tests uses a type called `ClientRunner` which represents the client-side of the conformance test suite and implements methods that are useful for the tests. The goal of the `ClientRunner` is to carry out high-level tasks that are useful to test the TUF specification, for example:
```

Comment:
Regardless of the typo, I'm having troubles parsing the paragraph. Is there a way to clarify, e.g. what "use a type" and "represent the client-side" means in this context?

Reply:
I wonder if we could just skip all of this (not talk about ClientRunner at all) and get directly to what client wrappers are for and what is required from them? It seems like writing a client wrapper is possible without knowing about ClientRunner?

Reply:
> Or alternatively reorder this so the client requirements come first and then there is a "technical details section?

I like this. It might be good to also include documentation for folks that want to contribute to the test suite, and for that the section about the `ClientRunner` is useful.

Reply:
Changed some things in https://github.com/theupdateframework/tuf-conformance/pull/30/commits/fe9266dcb4887c96d5bdaf9914ca703aad363b31.